server/src/main/java/org/elasticsearch/common/util/Maps.java

```diff
@@ -280,7 +280,7 @@ public static <K, V> Map<K, V> newMapWithExpectedSize(int expectedSize) {
      * @return a new pre-sized {@link HashMap}
      */
     public static <K, V> Map<K, V> newHashMapWithExpectedSize(int expectedSize) {
-        return new HashMap<>(capacity(expectedSize));
+        return HashMap.newHashMap(expectedSize);
```

Comment:
Since 19 both HashMap and LinkedHashMap could be presized using standard lib factories.

Reply:
nit: perhaps then (doesn't have to be here) we should convert all the uses of this helper to use HashMap.newHashMap directly?